.github/workflows/pr-review.yml

```
@@ -0,0 +1,56 @@
+
+  coderabbit:
+    name: CodeRabbit AI Review
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: CodeRabbit AI Reviewer
+        uses: coderabbitai/ai-pr-reviewer@latest
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)